pkg/spec/merge.go

```diff
@@ -204,7 +204,14 @@ func mergeParameter(parameter, parameter2 *spec.Parameter, path *field.Path) (*s
 	}
 
 	type1, type2 := parameter.Schema.Value.Type, parameter2.Schema.Value.Type
-	if type1 != type2 {
+	switch conflictSolver(type1, type2) {
+	case 0, 1:
```

Comment:
get we give an enum values instead of using 0,1,2...?

Reply:
we can, what will be the enum names?

Reply:
conflictType1, conflictType2, NoConflict, conflictUnresolved?

Reply:
**prefer**Type1, **prefer**Type2, NoConflict, conflictUnresolved?

Reply:
sure sounds good